cpp/test/CMakeLists.txt

```diff
@@ -222,6 +222,14 @@ if(BUILD_PRIMS_TESTS)
       ${COMMON_TEST_LINK_LIBRARIES}
   )
 
+  set_target_properties(
+    ${PRIMS_TEST_TARGET}
```

Comment:
shouldn't we also need a install target for `CUML_CPP_TEST_TARGET` and `CUML_MG_TEST_TARGET`?

Reply:
Yes we do need to install those as well

Reply:
addressed in latest commits :+1: